package.json

```diff
@@ -854,22 +855,22 @@
                     ],
                     "description": "The field values to display as labels in the treeview for Cosmos DB and MongoDB documents, in priority order"
                 },
-                "cosmosDB.showSavePrompt": {
+                "azureDatabases.showSavePrompt": {
                     "type": "boolean",
                     "default": true,
                     "description": "Show warning dialog when uploading a document to the cloud."
                 },
-                "cosmosDB.enableOutputTimestamps": {
+                "azureDatabases.enableOutputTimestamps": {
                     "type": "boolean",
                     "default": true,
                     "description": "Prepends each line displayed in the output channel with a timestamp."
                 },
-                "cosmosDB.emulator.mongoPort": {
+                "azureDatabases.emulator.mongoPort": {
```

Comment:
Changing setting names will break users with existing settings and changing command names will break users with existing keybindings. For settings/commands that are unique to Cosmos DB - I would really prefer not to change those